xds/src/main/java/io/grpc/xds/ClusterImplLoadBalancer.java

```diff
@@ -88,14 +88,19 @@
   private LoadBalancer childLb;
 
   ClusterImplLoadBalancer(Helper helper) {
-    this(helper, ThreadSafeRandomImpl.instance, TlsContextManagerImpl.getInstance());
+    this(helper, ThreadSafeRandomImpl.instance);
   }
 
+  ClusterImplLoadBalancer(Helper helper, ThreadSafeRandom random) {
```

Comment:
What's the point of creating this constructor?

Reply:
No reason! When I removed the call to `TlsContextManagerImpl.getInstance()`, I ended up with this 3 constructor refactor thinking only the last one is "test-only" constructor but this one is used in other places too. Removing it...